docs/submitting-applications.md

```diff
@@ -76,7 +76,7 @@ locally on your laptop), it is common to use `cluster` mode to minimize network
 the drivers and the executors. Currently, the standalone mode does not support cluster mode for Python
 applications.
 
-For Python applications, simply pass a `.py` file in the place of `<application-jar>` instead of a JAR,
+For Python applications, simply pass a `.py` file in the place of `<application-jar>`,
```

Comment:
I don't have a strong opinion though, this change looks okay. cc: @HyukjinKwon @srowen